azure-communication-ui/azure-communication-ui/src/main/res/drawable/azure_communication_ui_cancel_leave_call_button.xml

```diff
@@ -9,7 +9,7 @@
     <solid android:color="@color/azure_communication_ui_color_button_background_transparent" />
     <stroke
         android:width="2dp"
-        android:color="?attr/azure_communication_ui_calling_primary_color"
```

Comment:
Adam, this key is public and reviewed by Azure team. what com_ indicates?

Reply:
naming comes from fluent ui spec, I believe it stands for communication. I'll update documentation in the attrs, theme and custom theme files.